topcoffea/modules/remote_environment.py

```diff
@@ -31,7 +31,7 @@
         "packages": [
             f"python={py_version}",
             "pip",
-            "conda",
+            "conda=23.7.2",
```

Comment:
@anpicci is this necessary? I remember an issue when you were trying to get `lobster` to work, but do our coffea repos also not work with newer versions of conda?

Reply:
Actually, this is not strictly related neither with this PR nor with lobster, but with the MM thread reported [here](https://mattermost.web.cern.ch/topeft-runii/pl/67u1omnjb3b85pq9zz8ny666ya), sorry for not catching this. I propose to remove this pin from this PR, and include it in a new PR if necessary @bryates @kmohrman

Reply:
Ok, thanks for explaining. I agree it definitely sounds like this change should not go in with this PR.

Reply:
@anpicci looks like this was put back in. Can you unpin the conda version?

Reply:
you're right, sorry for the oversight